<launch>

  <include file="$(find soma_ros)/launch/orbslam2_rgbd.launch">
    <arg name="node_name" default="orbslam2_F"/>
    <arg name="color_image" default="/camera_F/color/image_raw"/>
    <arg name="depth_image" default="/camera_F/aligned_depth_to_color/image_raw"/>
    <arg name="camera_info" default="/camera_F/color/camera_info"/>
    <arg name="camera_frame_id" default="camera_F_link"/>
    <arg name="map_frame_id" default="orb_map_F"/>
  </include>

</launch>